pallets/capacity/src/tests/mock.rs

```diff
@@ -132,24 +130,40 @@ impl pallet_msa::Config for Test {
 	type MaxSignaturesStored = ConstU32<8000>;
 }
 
-// pub struct TestStakingRewardsProvider {}
-// impl StakingRewardsProvider<Test> for TestStakingRewardsProvider {
-// 	fn reward_pool_size(era: RewardEra) -> BalanceOf<Test> {
-// 		Balance::from(10_000u64);
-// 	}
-//
-// 	fn staking_reward_total(account_id: AccountId, from_era: RewardEra, to_era: RewardEra) -> BalanceOf<Test> {
-// 		Balance::from(10u64)
-// 	}
-//
-// 	fn validate_staking_reward_claim(account_id: AccountId, proof: Hash, payload: StakingRewardClaim<Test>) -> bool {
-// 		true
-// 	}
-//
-// 	fn payout_eligible(account_id: AccountId) -> bool {
-// 		true
-// 	}
-// }
+// not used yet
+pub struct TestStakingRewardsProvider {}
+
+type TestRewardEra = u32;
+
+impl StakingRewardsProvider<Test> for TestStakingRewardsProvider {
```

Comment:
maybe we can avoid these? Why do we need to mock this?

Reply:
For ease of testing, as I mention above. At some point the "real" trait will likely be much more complicated and depend on other stuff that we won't want to have to set up for every single test.